Makefile

```diff
@@ -319,6 +333,18 @@ create_cryostat_cr: destroy_cryostat_cr
 destroy_cryostat_cr:
 	- $(CLUSTER_CLIENT) delete -f config/samples/operator_v1beta1_cryostat.yaml
 
+# Build custom scorecard tests
+.PHONY: custom-scorecard-tests
+custom-scorecard-tests: fmt vet
```

Comment:
I am wondering if we should build this binary as a step in setting up the scorecard image in its `internal/images/custom-scorecard-tests/Dockerfile`? I think it would be consistent across dev environment?

Reply:
I'll change the Dockerfile to build the binary similar to the operator binary. Hopefully this will resolve Andrew's issue.